src/renderer/styles/global.css

```
@@ -91,6 +91,7 @@
 	--input: var(--border-strong);
 	--ring: var(--primary);
 	--radius: 0.5rem;
+	color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)